src/main/java/io/neonbee/internal/cluster/entity/UnregisterEntityVerticlesHook.java

```diff
@@ -0,0 +1,93 @@
+package io.neonbee.internal.cluster.entity;
+
+import static io.neonbee.hook.HookType.CLUSTER_NODE_ID;
+import static io.vertx.core.Future.succeededFuture;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.neonbee.NeonBee;
+import io.neonbee.hook.Hook;
+import io.neonbee.hook.HookContext;
+import io.neonbee.hook.HookType;
+import io.neonbee.internal.Registry;
+import io.neonbee.internal.cluster.ClusterHelper;
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+
+/**
+ * Hooks for unregistering verticle models.
+ */
+public class UnregisterEntityVerticlesHook {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(UnregisterEntityVerticlesHook.class);
```

Comment:
Just a personal preference, but would be nice to have all logs starting with the same case (either upper or lower). I think the majority is upper case.

Reply:
Good point, never paid attention to it.